File: machine-translation/google-cloud-translation/GoogleCloudTranslationMachineTranslation.java
// "input" is a string holding the source document.
// To translate from English to German, set "sourceLanguageCode" to "en" and "targetLanguageCode" to "de".

Translate translate = TranslateOptions.getDefaultInstance().getService();

Translation translation = translate.translate(
    input,
    Translate.TranslateOption.format("text"),
    Translate.TranslateOption.sourceLanguage(sourceLanguage),
    Translate.TranslateOption.targetLanguage(targetLanguage));

String translatedText = translation.getTranslatedText();

File: machine-translation/google-cloud-translation/GoogleCloudTranslationMachineTranslation.kt
// input a String containing the input document.
// "format" is either set to "text" or "html".
// To translate from English to German, set "sourceLanguage" to "en" and "targetLanguage" to "en".

val translate = TranslateOptions.getDefaultInstance().service

val translation = translate.translate(
    input,
    Translate.TranslateOption.format(format),
    Translate.TranslateOption.sourceLanguage(sourceLanguage),
    Translate.TranslateOption.targetLanguage(targetLanguage))

val translatedText = translation.translatedText
